features/support/object_model_pages/your_information.rb

```diff
@@ -27,4 +27,8 @@ def self.welcome_sign_in_message
   def self.continue_btn
     '.interaction-click-control-continue'
   end
-end
\ No newline at end of file
+
+  def self.interactive_verifications_page
+    "Upload document(s) proving your identity"
```

Comment:
this is still hard-coded english text -- needs to be a css class/attribute or reference a translation

Reply:
for this we don't have any specific css class/attribute except ul, li. and we don't have translation reference for this